Integrations/SyslogSender/SyslogSender.yml

```diff
@@ -0,0 +1,122 @@
+category: Analytics & SIEM
+commonfields:
+  id: Syslog Sender
+  version: -1
+configuration:
+- display: IP Address (e.g., 127.0.0.1)
+  name: address
+  required: true
+  type: 0
+- defaultvalue: '514'
+  display: Port
+  name: port
+  required: true
+  type: 0
+- defaultvalue: UDP
+  display: Protocol (TCP / UDP)
+  name: protocol
+  required: true
+  type: 0
+- defaultvalue: Low
+  display: Minimum severity of messages to send
+  name: severity
+  options:
+  - Unknown
+  - Low
+  - Medium
+  - High
+  - Critical
+  required: false
+  type: 15
+- defaultvalue: INFO
+  display: Log level to send
+  name: logging_level
+  options:
+  - DEBUG
+  - INFO
+  - WARNING
+  - ERROR
+  - CRITICAL
+  required: false
+  type: 15
+- defaultvalue: 'true'
+  display: Long running instance. Required for investigation mirroring and direct
+    messages.
+  name: longRunning
+  required: false
+  type: 8
+description: Use the Syslog Sender integration to send messages to Syslog.
+display: Syslog Sender
+name: Syslog Sender
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      default: true
+      defaultValue: all
+      description: The mirroring type. Can be "all", which mirrors everything, "chat",
+        which mirrors only chats (not commands), or "none", which stops all mirroring.
+      isArray: false
+      name: type
+      predefined:
+      - all
+      - chat
+      - none
+      required: false
+      secret: false
+    deprecated: false
+    description: Mirrors the investigation's War Room to Syslog.
+    execution: false
+    name: mirror-investigation
+  - arguments:
+    - default: true
+      description: The message content.
+      isArray: false
+      name: message
+      required: false
+      secret: false
+    - default: false
+      description: An entry ID to send as a link.
+      isArray: false
+      name: entry
```

Comment:
```suggestion
      name: entry_id
```

Reply:
this comes also from the server, not sure if we want to have 2 different args